lib/ansible/modules/windows/win_timezone.ps1

```diff
@@ -19,16 +19,15 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
-$result = @{
-    changed = $false
-    win_timezone = @{}
-}
-
 $params = Parse-Args $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
 
 $timezone = Get-AnsibleParam -obj $params -name "timezone" -failifempty $true -resultobj $result
 
+$result = @{
```

Comment:
Why oh why. I never really understood the purpose of the -resultobj option. Especially in this case.